src/elements/TextInput.tsx

```diff
@@ -75,7 +75,7 @@ export const TextInput: ForwardRefRenderFunction<
     startAdornment,
     endAdornment,
     disabled = false,
-    isOptional = false,
+    isOptional = true,
```

Comment:
Just to make sure this change gets incorporated fully, I noticed the use with setting `TextInputField` and `PasswordInputField` with `isOptional` since it extends `TextInputProps`, will it add value as part of this change if you add a `false` default within the `PasswordInputField.tsx`?
Or do you see it being set within each individual page. My thought is that if there is a PasswordInputField on the page it will always be required, and by default `false` for `isOptional`. I understand that it may get convoluted per set defaults in your current structure.

With all that, if you follow current pattern, when this is merged in we will need to incorporate it as part of the Settings.tsx page as well.

Reply:
Very good point, it doesnt hurt to set the default to false in the password field, if we need to override it anywhere to optional we can pass is the true value